src/spec_procs.cpp

```diff
@@ -6062,12 +6062,13 @@ SPECIAL(axehead) {
   int message_num;
   const char *axehead_messages[] = {
     "Runners these days don't realize how valuable keeping notes on their pocket secretary is. Like where Johnsons hang out.",
-    "Stick your radio and phone in a pocket. You can still hear 'em, and it keeps your hands free.",
-    "Seems like every day I hear about another wanna-be runner getting gunned down by the Star for walking around with their gun in their hand.",
+    "Don't hide your radio or phone in a box, you won't be able to hear them - but putting it right in your coat pocket won't do much but break it if you get shot.",
```

Comment:
Maybe "Best to keep your phone and radio in hand. In a box, you won't hear it, and in your pocket, it'll just get shot."